### PR TITLE
footer icons hover animation + fix

### LINKED ISSUE #129: Footer Hover Animation + Fixes
image

- This doesn't require tailwind animate.
- When the user hovers on the icons, the icons should scale up or translate up (you can choose either or).

Sample:

saf.ucrhighlanders.org
isa.ucrhighlanders.org

Fix:
- add shadow to the footer (same as the navigation bar)

### PR DESCRIPTION
**Pull request by xwRongx**
image

### CODE REVIEW

File: src/components/Footer.jsx [hunk 1]
```diff
@@ -3,12 +3,12 @@ import Link from "next/link";
 
 const Footer = () => {
   return (
-    <div className="flex items-center p-4 bg-biscuits-peach justify-between text-biscuits-purple-200 font-omc text-2xl md:text-3xl text-center">
+    <div className="flex shadow-[0_-2px_3px_0px_rgba(0,0,0,0.1)] items-center p-4 bg-biscuits-peach justify-between text-biscuits-purple-200 font-omc text-2xl md:text-3xl text-center">
```
Comment (seanquiambao) [inline src/components/Footer.jsx]: you can use tailwind's default drop shadow instead.

File: src/components/Footer.jsx [hunk 2]
```diff
@@ -3,12 +3,12 @@ import Link from "next/link";
...
       <div className="hidden md:flex tracking-widest"> H4B at UCR </div>
       <div className="flex md:hidden tracking-widest"> H4B </div>
-      <div className="flex flex-row space-x-2 md:space-x-5">
+      <div className="flex flex-row space-x-2 md:space-x-5 ">
         {FOOTER.map((social, index) => (
-          <Link key={index} href={social.link}>
+          <Link className="hover:-translate-y-1" key={index} href={social.link}>
```
Comment (seanquiambao) [inline src/components/Footer.jsx]: add a transition to the hover

**Review by seanquiambao — CHANGES_REQUESTED** [review]

**Comment by xwRongx** [comment]
image
With the default drop shadow, the footer shadow is different from the header shadow. Is it ok that they don't match?

**Comment by seanquiambao** [comment]
that should be fine

**Review by seanquiambao — APPROVED** [review]

**Pull request merged**